src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -457,7 +457,8 @@ func (c controller) applyPostInstallManifests() bool {
 	defer os.RemoveAll(tempDir)
 
 	customManifestPath := path.Join(tempDir, customManifestsFile)
-	if err := c.ic.DownloadFile(ctx, customManifestsFile, customManifestPath); err != nil {
+	err = c.ic.DownloadFile(ctx, customManifestsFile, customManifestPath)
```

Comment:
err is already defined in `tempDir, err := ioutil.TempDir("", "controller-custom-manifests-")` 
```suggestion
	if err = c.ic.DownloadFile(ctx, customManifestsFile, customManifestPath); err != nil {
```

Reply:
That code creates the error with the linter.

Reply:
And err is also already defined on line 470.. do we have a way we want to handle this as a team?